hack/get-project-maintainers.sh

```diff
@@ -19,8 +19,9 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck source=hack/common-vars.sh
+source "${REPO_ROOT}/hack/common-vars.sh"
 
-YQ="${REPO_ROOT}/hack/tools/bin/yq"
```

Comment:
Strange.. 
Looks like you missed adding `YQ` to `common-vars.sh` file?

Reply:
Oh! even all tests got passed

Reply:
Yep. The tests passed because I don't think we are promoting any images via this PR. 
[hack/get-project-maintainers.sh](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2523/files#diff-3f679eb4e9c5f17a0cf588ef16277a3ed735689d903ba80b898ffeaa2cbda741) is used to fetch the maintainers aliases, prefilled as a variable in the Makefile and used as part of `make promote-images`